### PR TITLE
Refactor-nav

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: components/CustomeSideBarMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import {
+	DrawerContentScrollView,
+	DrawerItemList,
+	DrawerItem,
+} from '@react-navigation/drawer';
+import SVGImage from '../assets/svgs/splash.svg';
+import { Feather } from '@expo/vector-icons';
+
+const CustomSidebarMenu = (props: any) => {
+	const BASE_PATH =
```
Comment (teebarg) [inline components/CustomeSideBarMenu.tsx]: are we using this?

File: components/CustomeSideBarMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		'https://raw.githubusercontent.com/AboutReact/sampleresource/master/';
+	const proileImage = 'react_logo.png';
```
Comment (teebarg) [inline components/CustomeSideBarMenu.tsx]: are we using this?

File: components/CustomeSideBarMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      style={{ flex: 1, marginTop: 30, paddingLeft: 5, paddingTop: 30 }}
+    >
+      {/*Top Large Image */}
+      <SVGImage width={150} height={75} style={{ marginLeft: 50 }} />
+      <DrawerContentScrollView {...props}>
+        <DrawerItemList {...props}/>
+        <DrawerItem
+          label={() => <Text style={{ color: "#9C9696" }}>Log Out</Text>}
+          icon={() => <Feather name="log-out" size={24} color="#9C9696" />}
+          onPress={() => Linking.openURL("https://aboutreact.com/")}
```
Comment (teebarg) [inline components/CustomeSideBarMenu.tsx]: About react?

File: screens/History.tsx [hunk 4]
```diff
@@ -69,60 +73,128 @@ export default function History({ navigation, route }) {
...
+                }}
+              >
+                E-LOAN ₦60,000{" "}
+              </Text>
+              <Text style={{ color: "#000", fontSize: 12 }}>
+                Order ID: 36584AG6
+              </Text>
+            </View>
+          </View>
+          <Text style={{ color: "#000", fontSize: 13 }}>12/22/2022</Text>
```
Comment (teebarg) [inline screens/History.tsx:93]: why are we hard coding this?

File: screens/History.tsx [hunk 5]
```diff
@@ -69,60 +73,128 @@ export default function History({ navigation, route }) {
...
+                }}
+              >
+                RENTAL LOAN ₦60,000
+              </Text>
+              <Text style={{ color: "#000", fontSize: 12 }}>
+                Order ID: 36584AG6
+              </Text>
+            </View>
+          </View>
+          <Text style={{ color: "#000", fontSize: 13 }}>12/22/2022</Text>
```
Comment (teebarg) [inline screens/History.tsx:112]: and this?

File: screens/History.tsx [hunk 6]
```diff
@@ -69,60 +73,128 @@ export default function History({ navigation, route }) {
...
+                }}
+              >
+                INFINIX HOT 10 LITE
+              </Text>
+              <Text style={{ color: "#000", fontSize: 12 }}>
+                Order ID: 36584AG6
+              </Text>
+            </View>
+          </View>
+          <Text style={{ color: "#000", fontSize: 13 }}>12/22/2022</Text>
```
Comment (teebarg) [inline screens/History.tsx:131]: and this

File: screens/History.tsx (hunk 4 above)
Comment (darmhoo) [reply screens/History.tsx:93]: Just a place holder. It will be my task for today

File: screens/History.tsx (hunk 5 above)
Comment (darmhoo) [reply screens/History.tsx:112]: same here

**Review by teebarg — APPROVED** [review]

**Pull request merged**